packages/connection/src/Client.php

```diff
@@ -131,7 +131,7 @@ public static function remote_request( $args, $body = null ) {
 		}
 
 		$url = add_query_arg( urlencode_deep( $url_args ), $args['url'] );
-		$url = \Jetpack::fix_url_for_bad_hosts( $url );
+		$url = Manager::fix_url_for_bad_hosts( $url );
```

Comment:
The fact that we're accessing the method from a static method somewhere else is another indication why it shouldn't live in the `Manager`.